include/mpi_support.h

```diff
@@ -2,11 +2,14 @@
 
 #include <mpi.h>
 
+#include "types.h"
```

Comment:
We should move the constant definition up to types.h (and out of the `mpi_support` namespace) since it has nothing to do with MPI.

Reply:
I will move it to `types.h` but on one hand, I don't fully agree that it has "nothing to do with MPI", on the other, since it is not a type, maybe we should have a separate header for all the constants.